code/web/src/modules/user/api/actions.js

```diff
@@ -103,11 +104,42 @@ export function logoutUnsetUserLocalStorageAndCookie() {
   // Remove token
   window.localStorage.removeItem('token')
   window.localStorage.removeItem('user')
-
+  
   // Remove cookie
   cookie.remove('auth')
 }
 
+export function fetchStylePreference(userId) {
```

Comment:
It was really interesting to walk through writing these fetchStyle and setStyle functions.  I feel as though I have a decent grasp of their purpose and what is happening within each.  Thanks for that, Kim and Kyle!